skyportal/handlers/comment.py

```diff
@@ -135,11 +135,20 @@ def delete(self, comment_id):
               application/json:
                 schema: Success
         """
-        # TODO: Check ownership
-        comment = Comment.query.get(comment_id)
-        DBSession().delete(comment)
-        DBSession().commit()
-
+        user = (self.current_user.username if hasattr(self.current_user, 'username') else self.current_user.name)
```

Comment:
Is this because tokens store their username on a different variable?  Any reason we can't just name them the same?